testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -126,6 +126,16 @@ Feature: Smoke tests for <client>
     Then I should see a "Channel Subscriptions successfully changed for" text
     And I reboot the "<client>" if it is a transactional system
```

Comment:
We do a reboot of SLE micro clients here, so I'm not sure having another scenario to reboot again immediately after this one is needed.

Reply:
Yes, that what I meant by do we really need to have a specific feature to reboot slemicro because we reboot it every changes.